README.md

```diff
@@ -101,14 +101,188 @@ These can be used in the `FromBauble` macro, and can be automatically implemente
 ```rust
 #[derive(FromBauble)]
 struct Button {
-  #[bauble(attribute)]
-  width: u32,
-  #[bauble(attribute)]
-  height: u32,
-  on_press: Action,
+    #[bauble(attribute)]
+    width: u32,
+    #[bauble(attribute)]
+    height: u32,
+    on_press: Action,
 }
 ```
 
+## `FromBauble` derive macro attributes
+
+### `attribute`
+
+See above.
+
+### `default`
+
+```rust
+// Rust
+#[derive(FromBauble)]
+struct Background {
+    content: Box<Widget>,
+    #[bauble(default = Color::Black)]
+    color: Color,
+    #[bauble(default)]
+    margin: Margin,
+}
+
+// bauble
+background = Background {
+    content: Widget::Text(Text("Hello World")),
+}
+```
+
+On a field, allows it to be omitted and replaced with a default expression. If no expression is
+provided, that field's `Default` implementation will be used.
+
+### `from_default`
```

Comment:
I'm having trouble finding where this is implemented in the derive macro. Could you point that out to me?

Reply:
Whoops, that was supposed to be `as_default` :sweat_smile: